### PR TITLE
Add group flag

### PR DESCRIPTION
**Pull request by nsams**
depends on #17

this allows annotating scripts with groups and starting group of scripts

I will add Changelogs and Docs later as there is more planned

### CODE REVIEW

File: src/commands/start.command.ts [hunk 1]
```diff
@@ -6,27 +6,25 @@ import { createServer, Socket } from "net";
...
-                console.error(`${colors.bgRed.bold.black(" dev-pm ")} Script ${scriptName} not found in dev-pm config`);
-                process.exit(1);
-            }
-        });
-    } else {
-        scriptsToStart = scripts;
-    }
+    const scriptsToStart: ScriptDefinition[] = scripts.filter((script) => {
+        if (options?.only && options.only.length > 0) {
+            if (!options.only.includes(script.name)) return false;
```
Comment (Fabian-Fynn) [inline src/commands/start.command.ts:17]: Why don't we log if a script is not found?

File: src/commands/start.command.ts [hunk 2]
```diff
@@ -6,27 +6,25 @@ import { createServer, Socket } from "net";
...
+        }
+        if (options?.onlyGroup && options.onlyGroup.length > 0) {
```
Comment (Fabian-Fynn) [inline src/commands/start.command.ts:19]: Is it on purpose that both only and onlyGroup options can be combined?

Comment (nsams) [reply src/commands/start.command.ts:19]: yes, why not

File: src/commands/start.command.ts (hunk 1 above)
Comment (nsams) [reply src/commands/start.command.ts:17]: good idea, done in https://github.com/vivid-planet/dev-process-manager/pull/28

**Comment by nsams** [comment]
@Fabian-Fynn please approve

**Review by Fabian-Fynn — APPROVED** [review]

**Pull request merged**